optuna/integration/tensorboard.py

```diff
@@ -56,34 +56,44 @@ def __call__(self, study: optuna.study.Study, trial: optuna.trial.FrozenTrial) -
     def _add_distributions(
         self, distributions: Dict[str, optuna.distributions.BaseDistribution]
     ) -> None:
+        real_distributions = (
+            optuna.distributions.UniformDistribution,
+            optuna.distributions.LogUniformDistribution,
+        )
+        discrete_distributions = (
+            optuna.distributions.UniformDistribution,
+            optuna.distributions.LogUniformDistribution,
```

Comment:
Shouldn't we remove these distributions from `discrete_distributions`?

Reply:
Ah, thank you so much for the catch. c6b9994 removed it.